### PR TITLE
Add note about some cameras needing to seek to get a valid snapshot

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Add note about some cameras needing to seek to get a valid snapshot

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/onvif.markdown [hunk 1]
```diff
@@ -46,6 +46,10 @@ You can configure specific FFmpeg options through the integration options flow b
 | Extra FFmpeg arguments | Extra options to pass to `ffmpeg`, e.g., image quality or video filter options. More details in [`ffmpeg` integration](/integrations/ffmpeg). |
 | Use wallclock as timestamps | ([Advanced Mode](/blog/2019/07/17/release-96/#advanced-mode) only) Rewrite the camera timestamps. This may help with playback or crashing issues from Wi-Fi cameras or cameras of certain brands (e.g., EZVIZ). |
 
+#### Snapshots
+
+Some cameras will not produce usable snapshots with larger stream sizes and may require seeking in the stream to get a usable snapshot. Setting the `Extra FFmpeg arguments` to `-pred 1 -ss 00:00:05 -frames:v 1` will cause the snapshot to be taken 5 seconds into the stream.
```
Comment (bdraco) [inline source/_integrations/onvif.markdown]:
```suggestion
Some cameras will not produce usable snapshots with larger stream sizes.

By default the integration will only enable the camera entity for the first H264 profile. If you are unable to get working snapshot:

- If there are additional camera entities available for other profiles, try enabling those entities.
- Set the `Extra FFmpeg arguments` to `-pred 1 -ss 00:00:05 -frames:v 1` to cause the snapshot to be taken 5 seconds into the stream.
```

File: source/_integrations/onvif.markdown [hunk 2]
```diff
@@ -46,6 +46,15 @@ You can configure specific FFmpeg options through the integration options flow b
 | Extra FFmpeg arguments | Extra options to pass to `ffmpeg`, e.g., image quality or video filter options. More details in [`ffmpeg` integration](/integrations/ffmpeg). |
 | Use wallclock as timestamps | ([Advanced Mode](/blog/2019/07/17/release-96/#advanced-mode) only) Rewrite the camera timestamps. This may help with playback or crashing issues from Wi-Fi cameras or cameras of certain brands (e.g., EZVIZ). |
 
+#### Snapshots
+
+Some cameras will not produce usable snapshots with larger stream sizes.
+
+By default the integration will only enable the camera entity for the first H264 profile. If you are unable to get working snapshot:
+
+- If there are additional camera entities available for other profiles, try enabling those entities.
```
Comment (bdraco) [inline source/_integrations/onvif.markdown]:
```suggestion
- If additional camera entities are available for other profiles, try enabling those entities.
```

File: source/_integrations/onvif.markdown [hunk 3]
```diff
@@ -46,6 +46,15 @@ You can configure specific FFmpeg options through the integration options flow b
 | Extra FFmpeg arguments | Extra options to pass to `ffmpeg`, e.g., image quality or video filter options. More details in [`ffmpeg` integration](/integrations/ffmpeg). |
 | Use wallclock as timestamps | ([Advanced Mode](/blog/2019/07/17/release-96/#advanced-mode) only) Rewrite the camera timestamps. This may help with playback or crashing issues from Wi-Fi cameras or cameras of certain brands (e.g., EZVIZ). |
 
+#### Snapshots
+
+Some cameras will not produce usable snapshots with larger stream sizes.
+
+By default the integration will only enable the camera entity for the first H264 profile. If you are unable to get working snapshot:
```
Comment (bdraco) [inline source/_integrations/onvif.markdown]:
```suggestion
By default, the integration will only enable the camera entity for the first H264 profile. If you are unable to get a working snapshot:
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @bdraco :+1:

**Comment by bdraco** [comment]
Thanks

**Pull request merged**